it.json

```diff
@@ -0,0 +1,222 @@
+{
+    "app.shelly-home.more.language[en].label" : "English",
+    "app.shelly-home.more.language[de].label" : "Deutsch",
+    "app.shelly-home.more.language[nl].label" : "Nederlands",
+    "app.shelly-home.more.language[es].label" : "Español",
+    "app.shelly-home.more.language[es].label" : "Italiano",
```

Comment:
Please change `es` to `it` here

Also in the i18n.model.ts file you need to add `it`

Reply:
ok i do it